src/helper/helper.js

```diff
@@ -310,20 +318,31 @@ function buildAbstractModelInterface_(dataLayerHelper) {
  * @private
  */
 function processCommand_(command, model) {
-  if (!isString_(command[0])) return;
+  if (!isString_(command[0])) {
+    logError(`Error processing command, no command was run. The first ` +
+        `argument must be of type string, but was of type ` +
+        `${typeof command[0]}. \nYou pushed the command\n ${command}.`,
+        LogLevel.WARNING);
+  }
   const path = command[0].split('.');
   const method = path.pop();
   const args = command.slice(1);
   let target = model;
   for (let i = 0; i < path.length; i++) {
-    if (target[path[i]] === undefined) return;
+    if (target[path[i]] === undefined) {
+      logError(`The object at \n${path}\n was undefined, so the` +
```

Comment:
I'd probably reverse this on the same prinicple of "effect then cause" actionable messages, i.e.:

The command was ignored because the path was undefined.